cmd/kubeadm/app/cmd/phases/addons_test.go

```diff
@@ -20,7 +20,6 @@ import (
 	"testing"
 
 	// required for triggering api machinery startup when running unit tests
-	_ "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/install"
 
```

Comment:
Delete comment as well? (Same applies to other test file in the same folder

Reply:
Done